services/api.py

```diff
@@ -1142,6 +1142,11 @@ def validate_service_node_ids(service_node_ids):
                 arg = filters["address"] + r"($|\s|,|[a-zA-Z]).*"
             queryset = queryset.filter(**{key: arg})
 
+        if "no_private_services" in filters:
+            queryset = queryset.exclude(
+                Q(displayed_service_owner_type="PRIVATE_SERVICE") | Q(organizer_type=10)
```

Comment:
It would be more explicit use enums or constants for these.
Eg. `OWNER_TYPE_PRIVATE_SERVICE = "PRIVATE_SERVICE"` and `ORGANIZER_TYPE_PRIVATE_ENTERPRISE = 10`

Reply:
I agree. The choices for the `ORGANIZER_TYPES` was implemented in such way that e.g. `ORGANIZER_TYPES.PRIVATE_ENTERPRISE` was not possible but using a constant would make this piece a bit more readable. I'll do some changes 👍

Reply:
Updated now.